docs/Open Data Model/Open Data Model.md

```diff
@@ -0,0 +1,755 @@
+“dst_host”:”test3.companyA.com”,
+“dst_port”:443,
+“aproto”:”sshd”
+“msg”:”user:mhicks successfully logged in  to test3.companyA.com from 192.168.1.1”,
+“username”:”mhicks”,
+}
+
+
+**Parquet**
+-------
+
+Parquet is a columnar storage format that offers the benefits of compression and efficient columnar data representation and is optimal for batch analytic use cases.  More information on parquet can be found here:  https://parquet.apache.org/documentation/latest/ 
+It should be noted that conversion from Avro to Parquet is supported. This allows for data collected and analyzed for stream-based use cases to be easily converted to Parquet for longer-term batch analytics.
+
+**Example - Advanced Threat Modeling**
+----------------------------------
+
+In this example, the ODM is leveraged to build an “event” table for a threat model that uses attributes native to the ODM and derived attributes, which are calculations based on the aggregate data stored in the model.  In this context, an “event” table is defined by the attributes to be evaluated for predictive power in identifying threats and the actual attribute values (i.e rows in the table).  In the example below, the event table is composed of the following attributes, which are then leveraged to identify threats via a Risk Score analytic model:
+
+ - “src_ipv4” - This attribute is native to the security event log component of the ODM and represents the source IP address of the corresponding table row
+ - “os” - This attribute is native to the endpoint context component of the ODM and represents the operating system of the endpoint system in the table row
+ - SUM (in_bytes + out_bytes) for the last 7 days - “in_bytes” and “out_bytes” are native to the security event log component of the ODM.  This derived attribute represents a summation of bytes between the source address and destination domain over the last 7 days
+ - “dst_domain” - This attribute is native to the security event log component of the ODM and represents the destination domain 
+ - Days since “creation_date” - “creation_date” is native to the network context component of the ODM and represents the date the referenced domain was registered. This derived attribute calculates the days since the domain was created/registered.
+
+|**src_ipv4**|**os**|**dst domain**|**Days since “creation_date”**|**SUM (in_bytes + out_bytes)**|**Risk Score (1-100)**|
+|---|---|---|---|---|---|
+|10.1.1.10|Microsoft|dajdkwk.com|39|3021 MB|99
+|192.168.8.9|Redhat|usatoday.com|3027|2 MB|2
+|172.16.32.3|Apple|box.com|1532|76 MB|10
+|192.168.4.4|Microsoft|kzjkeljr.ru|3|0.9 MB|92
+
+
+The **“Risk Score”**  attribute represents potential output from a threat detection model based on the attributes and values represented in the “event” table and is provided as an example of what is enabled by the ODM.  Can you tell which attributes and values hold predictive power for threat detection?
+
+**Example - Singular Data View for Complete Context**
```

Comment:
This heading should have an underline as well, otherwise it appears smaller than the previous example heading.